crates/holochain_metrics/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "holochain_metrics"
+version = "0.3.0-beta-dev.1"
+authors = ["Holochain Core Dev Team <devcore@holochain.org>"]
+edition = "2021"
+description = "metrics helpers"
+license = "MIT"
+documentation = "https://docs.rs/holochain_metrics"
+repository = "https://github.com/holochain/holochain"
+
+[features]
+default = []
+
+[dependencies]
+influxive-child-svc = "=0.0.1-alpha.2"
+influxive-otel = "=0.0.1-alpha.2"
```

Comment:
Can these be behind a feature flag for Influx so we can compile out different backends?

Reply:
Yep :+1: it's on my list to feature flipper this before merging.

Reply:
done.